drivers/adult_only_households_sub_pop/app/controllers/adult_only_households_sub_pop/dashboards/adult_only_households_controller.rb

```diff
@@ -7,8 +7,7 @@
 module AdultOnlyHouseholdsSubPop::Dashboards
   class AdultOnlyHouseholdsController < ::Dashboards::BaseController
     include ArelHelper
-
-    before_action :require_can_view_censuses!
+    include WarehouseReportAuthorization
```

Comment:
Does this need to be here if it is in the base class?

Reply:
Probably not